doc/man3/SSL_CONF_cmd.pod

```diff
@@ -441,6 +452,15 @@ middleboxes that do not understand TLSv1.3 will not drop the connection. This
 option is set by default. A future version of OpenSSL may not set this by
 default. Equivalent to B<SSL_OP_ENABLE_MIDDLEBOX_COMPAT>.
 
+B<AntiReplay>: If set then OpenSSL will automatically detect if a session ticket
+has been used more than once, TLSv1.3 has been neotiated and early data is
+enabled on the server. A full handshake is forced if a session ticket is used a
+second or subsequent time. This option is set by default and is only used by
+servers. Anti-replay measures are required to comply with the TLSv1.3
+specification. Some applications may be able to mitigate the replay risks in
+other ways and in such cases the built in OpenSSL functionality is not required.
+Equivalent to B<SSL_OP_NO_ANTI_REPLAY>.
```

Comment:
If it is equivalent to the `NO` option, the name is confusing.

Reply:
Indeed.

Reply:
It's actually equivalent to the inverse of the `NO` option. I clarified this.